composer.json

```
@@ -18,8 +18,9 @@
         "symfony/polyfill-php80": "^1.27"
     },
     "require-dev": {
-        "squizlabs/php_codesniffer": "^3.7",
         "doctrine/coding-standard": "^11.1",
+        "rector/rector": "^0.16.0",
```

```suggestion
        "rector/rector": "^0.16",
```

Is there any significance of the trailing `.0`? I don't recall offhand if `^` has different rules for pre-1.0 versions or not (where this might instead prohibit 0.17+).

---

Before 1.0, that's more safe to stay with `^0.16.0` which is equivalent to `>= 0.16.0 < 0.17.0`
https://getcomposer.org/doc/articles/versions.md#caret-version-range-
> For pre-1.0 versions it also acts with safety in mind and treats ^0.3 as >=0.3.0 <0.4.0 and ^0.0.3 as >=0.0.3 <0.0.4.

---

Aye, that's what I was thinking of. Thanks for confirming.